### PR TITLE
Uppdatering beskrivning Valberedningen

### PR DESCRIPTION
**Pull request by Blytungdev**
Uppdatering av den svenska sidan samt skapande av den engelska

## Describe your changes
Skrev en bättre beskrivning och upptäckte det engelska språket
## Checklist before requesting a review

- [x] I have performed a self-review of my changes
- [x] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

**Comment by Blytungdev** [comment]
@niklasvatn @elmaxe

File: en/clubs/Valberedningen/body.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# Under Construction
\ No newline at end of file
+# Nominating Committee (Valberedningen)
```
Comment (Herkarl) [inline en/clubs/Valberedningen/body.md:1]:
```suggestion
# Election Committee (Valberedningen)
```

File: en/clubs/Valberedningen/body.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+The Nominating Committee administrates elections within the chapter and arranges the Chapter Fair once per semester. The Nominating Committee usually consists of 6 Committee members and [The Chairperson](https://dfunkt.datasektionen.se/position/id/34/).
```
Comment (Herkarl) [inline en/clubs/Valberedningen/body.md:3]:
```suggestion
The Election Committee administrates elections within the chapter and arranges the Chapter Fair once per semester. The Election Committee usually consists of 6 Committee members and [The Chairperson](https://dfunkt.datasektionen.se/position/id/34/).
```

File: en/clubs/Valberedningen/body.md (hunk 1 above)
Comment (Herkarl) [inline en/clubs/Valberedningen/body.md:1]:
```suggestion
# Election Committee (Valberedningen)
```

File: en/clubs/Valberedningen/body.md [hunk 3]
```diff
@@ -1 +1,13 @@
...
+
+## Elections during Chapter Meetings
+
+Ahead of each election the Nominating Committee opens nominations for posts at [val.datasektionen.se](https://val.datasektionen.se/). Those nominated can then either accept or decline their nominations. When a candidate accepts their nomination the Nominating Committee will invite the candidate to an interview. Based on the interview, a written statement will be produced which the candidate can correct any errors in. Thereafter the Nominating Committee will make a judgement on whether the candidate is suitable for the post. All statements and judgements will be published in the election documents for the Chapter Meeting to review.
```
Comment (Herkarl) [inline en/clubs/Valberedningen/body.md:7]:
```suggestion
Ahead of each election the Election Committee opens nominations for posts at [val.datasektionen.se](https://val.datasektionen.se/). Those nominated can then either accept or decline their nominations. When a candidate accepts their nomination the Election Committee will invite the candidate to an interview. Based on the interview, a written statement will be produced which the candidate can correct any errors in. Thereafter the Election Committee will make a judgement on whether the candidate is suitable for the post. All statements and judgements will be published in the election documents for the Chapter Meeting to review.
```

File: en/clubs/Valberedningen/body.md [hunk 4]
```diff
@@ -1 +1,13 @@
...
+
+At the Chapter Meeting each candidate get to present themselves and answer questions. The Chapter Meeting will discuss the candidate and then decide their fate.
+
+## The Chapter Fair
+
+Once per semester the Nominating Committee arranges the Chapter Fair. At the Fair, chapter officials, committees and projects present their activities to the chapter members. There might also be a patch for those deemed worthy.
```
Comment (Herkarl) [inline en/clubs/Valberedningen/body.md:13]:
```suggestion
Once per semester the Election Committee arranges the Chapter Fair. At the Fair, chapter officials, committees and projects present their activities to the chapter members. There might also be a patch for those deemed worthy. 
```

File: en/clubs/Valberedningen/sidebar.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+## Contact
+
+#### Nominating Committee Chairperson
```
Comment (Herkarl) [inline en/clubs/Valberedningen/sidebar.md:3]:
```suggestion
#### Election Committee Chairperson
```

**Review by Herkarl — CHANGES_REQUESTED** [review]
Suggested adjustment of translation of valberedning according to previous conversation

File: en/clubs/Valberedningen/sidebar.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+Hannah Burak</br>
+[valberedning@d.kth.se](mailto:valberedning@d.kth.se)
```
Comment (Herkarl) [inline en/clubs/Valberedningen/sidebar.md:5]: This email should be [valberedning-ordf@d.kth.se](mailto:valberedning-ordf@d.kth.se)

[valberedning@d.kth.se](mailto:valberedning@d.kth.se) should probably still be included, perhaps just below ## Contact?

Same comments apply to Swedish page

**Review by Herkarl — CHANGES_REQUESTED** [review]
Otherwise lgtm

**Comment by Herkarl** [comment]
@Blytungdev @lucubrator

**Review by Herkarl — APPROVED** [review]

**Pull request merged**